flask_resty/view.py

```diff
@@ -477,18 +488,15 @@ def update(
         )
         data_in = self.get_request_data(expected_id=id, partial=partial)
 
-        self.update_item(item, data_in)
+        item = self.update_item(item, data_in) or item
         self.commit()
 
-        if return_content:
-            return self.make_item_response(item)
-
-        return self.make_empty_response(item=item)
+        return self.make_updated_response(item, return_content=return_content)
 
     def destroy(self, id):
         item = self.get_item_or_404(id)
 
-        self.delete_item(item)
+        item = self.delete_item(item) or item
```

Comment:
this `or item` will never be hit because delete_item always returns an item, since there is an `or item` also there. so you either delete it from here or there. same for delete

Reply:
we don't want overriders of `delete_item` to have to return an item